packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -244,9 +244,10 @@ export const connectEpic: Epic<AnyAction, AnyAction, any, any> = (action$, state
             const timeout$ = of(subscriptionError({ ...action, reason: 'WS: unsubscribe -> subscribe (unresponsive channel)' }), unsubscribe(action), subscribe(action)).pipe(
               delay(config.subscriptionUnresponsiveTTL),
               withLatestFrom(state$),
-              map(([action, state]) => ({ ...action, isActive: state.ws.subscriptions[subscriptionKey]?.active })),
+              map(([action, state]) => ({ ...action, isActive: state.ws.subscriptions[subscriptionKey]?.active || state.ws.subscriptions[subscriptionKey]?.subscribing > 0 })),
               // Filters by active subscription. 
               // The timeout could think we don't receive messages because of unresponsiveness, and it's actually unsubscribed
+              // Subscribing is considered active as we want to trigger an unsubscription from a hung subscribing channel
               filter(({ isActive }) => isActive),
               tap(() => logger.info('WS: unsubscribe -> subscribe (unresponsive channel)', { payload: action.subscriptionMsg })),
```

Comment:
Also, this `logger.info` will happen 3x (for every event), while we need it to happen only once. Do we need it, as I think we already log on `subscriptionError` in metrics, and it carries a `reason` string.